src/goto-programs/remove_function_pointers.cpp

```diff
@@ -348,6 +375,41 @@ void remove_function_pointerst::remove_function_pointer(
     }
   }
 
+  // we only handle cases where the LHS is a symbol or a member of a struct
+  if(pointer.id() == ID_symbol || pointer.id() == ID_member)
+  {
+    irep_idt identifier;
+    if(pointer.id() == ID_symbol)
+      identifier =
+        to_symbol_expr(to_dereference_expr(code.function()).pointer())
+          .get_identifier();
+    else
+    {
+      identifier = get_member_identifier(to_member_expr(pointer));
+    }
+
+    if(function_pointer_map.find(identifier) != function_pointer_map.end())
```

Comment:
Instead of `find` followed by `[]` use a construct like:
```
auto find_identifier = function_pointer_map.find(identifier);
if(find_identifier != function_pointer_map.end())
{
  functionst &map_entry = find_identifier->second;
  ...
```